cmd/helm/helm.go

```diff
@@ -68,8 +68,7 @@ func main() {
 	cmd := newRootCmd(actionConfig, os.Stdout, os.Args[1:])
 
 	if err := actionConfig.Init(settings.RESTClientGetter(), settings.Namespace(), os.Getenv("HELM_DRIVER"), debug); err != nil {
-		debug("%+v", err)
-		os.Exit(1)
+		log.Fatalf("%+v", err)
```

Comment:
Why use `%+v`? Should this be `%s`?

Reply:
I kept that from the previous way this error was being handled. The `%+v` is supposed to print out the error struct nicely, but because the error struct just contains a string, it makes no difference using `%+v` vs `%s`. In fact, we could avoid using `log.Fatalf` all together and use `log.Fatal(err)`. Let me know and I can change it to whichever seems best.

Reply:
`log.Fatal(err)` is the more idiomatic approach, so let's go ahead with that.

Reply:
Yep, I agree with `log.Fatal(err)` as well.